tests/e2e/testcases/blockchain/configure/suite.go

```diff
@@ -0,0 +1,470 @@
+		chainConfig := getBlockchainConfig(newRPCTxFeeCap1)
+		chainConfigPath, err := utils.CreateTmpFile(constants.ChainConfigFileName, []byte(chainConfig))
+		gomega.Expect(err).Should(gomega.BeNil())
+		_, err = commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"chain-config": chainConfigPath,
+			},
+		)
+		gomega.Expect(err).Should(gomega.BeNil())
+		// deploy l1
+		output, err := utils.TestCommand(
+			utils.BlockchainCmd,
+			"deploy",
+			[]string{utils.BlockchainName},
+			utils.GlobalFlags{},
+			utils.TestFlags{
+				"local":             true,
+				"skip-icm-deploy":   true,
+				"skip-update-check": true,
+			},
+		)
+		gomega.Expect(output).Should(gomega.ContainSubstring("L1 is successfully deployed on Local Network"))
+		gomega.Expect(err).Should(gomega.BeNil())
+		blockchainID, err := utils.ParseBlockchainIDFromOutput(output)
+		gomega.Expect(err).Should(gomega.BeNil())
+		nodesInfo, err := utils.GetLocalClusterNodesInfo()
+		gomega.Expect(err).Should(gomega.BeNil())
+		// check
+		checkBlockchainConfig(nodesInfo, blockchainID, newRPCTxFeeCap1, nil)
+		// stop
+		err = commands.StopNetwork()
+		gomega.Expect(err).Should(gomega.BeNil())
+		// cleanup logs
+		cleanupLogs(nodesInfo, blockchainID)
+		// change blockchain config
+		chainConfig = getBlockchainConfig(newRPCTxFeeCap2)
+		chainConfigPath, err = utils.CreateTmpFile(constants.ChainConfigFileName, []byte(chainConfig))
+		gomega.Expect(err).Should(gomega.BeNil())
+		_, err = commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"chain-config": chainConfigPath,
+			},
+		)
+		gomega.Expect(err).Should(gomega.BeNil())
+		// start
+		out := commands.StartNetwork()
+		gomega.Expect(out).Should(gomega.ContainSubstring("Network ready to use"))
+		// check
+		checkBlockchainConfig(nodesInfo, blockchainID, newRPCTxFeeCap2, nil)
+	})
+
+	ginkgo.It("set per node blockchain config", func() {
+		// set per node blockchain config before deploy
+		nodesRPCTxFeeCap := map[string]int{
+			node1ID: newRPCTxFeeCap1,
+			node2ID: newRPCTxFeeCap2,
+		}
+		perNodeChainConfig := getPerNodeChainConfig(nodesRPCTxFeeCap)
+		perNodeChainConfigPath, err := utils.CreateTmpFile(constants.PerNodeChainConfigFileName, []byte(perNodeChainConfig))
+		gomega.Expect(err).Should(gomega.BeNil())
+		_, err = commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"per-node-chain-config": perNodeChainConfigPath,
+			},
+		)
+		gomega.Expect(err).Should(gomega.BeNil())
+		// deploy l1
+		output, err := utils.TestCommand(
+			utils.BlockchainCmd,
+			"deploy",
+			[]string{utils.BlockchainName},
+			utils.GlobalFlags{},
+			utils.TestFlags{
+				"local":                   true,
+				"num-local-nodes":         2,
+				"staking-cert-key-path":   node1CertPath + "," + node2CertPath,
+				"staking-tls-key-path":    node1TLSPath + "," + node2TLSPath,
+				"staking-signer-key-path": node1BLSPath + "," + node2BLSPath,
```

Comment:
Seems that this test only modifies the rpc tx fee, why do we need

```
"staking-cert-key-path":   node1CertPath + "," + node2CertPath,
"staking-tls-key-path":    node1TLSPath + "," + node2TLSPath,
"staking-signer-key-path": node1BLSPath + "," + node2BLSPath,
```

Reply:
CLI accepts specifying blockchain configs on a per node basis. For that, CLI needs the node ID associated to each confg.
So, the test needs to start the L1 with predefined node IDs for the validators.